streams/src/main/java/org/apache/kafka/streams/kstream/JoinWindows.java

```diff
@@ -31,7 +31,9 @@
  *     WHERE
  *       stream1.key = stream2.key
  *       AND
- *       stream2.ts - before <= stream1.ts <= stream2.ts + after
+ *       stream2.ts >= stream1.ts - before
+ *       AND
+ *       stream2.ts <= stream1.ts + after
```

Comment:
It's not about having valid SQL here IMHO.
Anyway, I would at least reverse order and make a single line (right now, I have to thing twice to parse the code), or we could also use `BETWEEN`:
```
stream1.ts - before <= stream2.ts AND stream2.ts <= stream1.ts + after
```
or
```
stream2.ts BETWEEN stream1.ts - before AND stream1.ts + after
```

WDYT?